### PR TITLE
Example metadata: expressions.md

### PR DESCRIPTION
**Pull request by RexJaeschke**

### CODE REVIEW

**Review by jskeet — APPROVED** [review]

**Comment by jskeet** [comment]
Aargh - have merged this thinking it was a single whitespace change (having followed a link from GitHub which only showed me a single commit). Will now review all the changes instead...

File: standard/expressions.md [hunk 1]
```diff
@@ -4811,24 +4808,14 @@ It is possible for anonymous function delegates to share some captured variables
...
-> the three delegates capture the same instance of `x` but separate instances of `y`, and the output is:
->
-> ```console
-> 1 1
-> 2 1
-> 3 1
-> ```
->
-> *end example*
+> the three delegates capture the same instance of `x` but separate instances of `y`. *end example*
```
Comment (jskeet) [inline standard/expressions.md:4811]: Why was this change made? I really think we want the output here - and for it to be tested.

**Review by jskeet — COMMENTED** [review]
Just one additional comment having reviewed the rest of the PR.

File: standard/expressions.md (hunk 1 above)
Comment (RexJaeschke) [reply standard/expressions.md:4811]: @jskeet As the example was written, while it captured the delegates in an array of 3 elements, it never actually executed those functions, so no output was produced. What I will do in a separate PR is to add a loop that executes the functions, so output is actually produced, and we can check that.

Comment (jskeet) [reply standard/expressions.md:4811]: Sounds good to me.

**Pull request merged**